packages/excalidraw/fonts/FontMetadata.ts

```diff
@@ -147,3 +147,14 @@ export const GOOGLE_FONTS_RANGES = {
 
 /** local protocol to skip the local font from registering or inlining */
 export const LOCAL_FONT_PROTOCOL = "local:";
+
+export const loadFallbackFont = (fontFamily: string) => {
+  if (fontFamily in FONT_FAMILY_FALLBACKS && !document.fonts.check(`12px ${fontFamily}`)) {
+    const link = document.createElement('link');
+    link.href = `https://fonts.googleapis.com/css2?family=${fontFamily.replace(' ', '+')}&display=swap`;
+    link.rel = 'stylesheet';
+    document.head.appendChild(link);
+  }
+};
+
+loadFallbackFont('Xiaolai');
```

Comment:
Hi, this won't work as:

1. Xiaolai is self-hosted, not on Google CDN.
2. We can't rely solely on DOM APIs, as it would break our server-side use cases (i.e. `@excalidraw/utils`).
3. We rely on FontFace API for most of our fonts.

The task is about lazy loading the font faces, though technically it might be done from different perspectives and I am not even sure yet which one is the best.

Reply:
Thanks for the feedback! @Mrazator 
Given that Xiaolai is self-hosted and the use of DOM APIs can break server-side use cases, should I work around this by fully utilizing the FontFace API for lazy loading the fonts? I can shift the logic to handle font loading using the FontFace API to ensure it’s compatible with SSR and client-side environments.

Would you prefer I explore this approach further, or is there another direction you'd suggest for implementing the lazy loading?